lib/foreman_maintain/package_manager/apt.rb

```diff
@@ -2,7 +2,15 @@ module ForemanMaintain::PackageManager
   class Apt < Base
     def installed?(packages)
       packages_list = [packages].flatten(1).map { |pkg| "'#{pkg}'" }.join(' ')
-      sys.execute?(%(dpkg-query -l #{packages_list}))
+      status, output = sys.execute_with_status(%(dpkg --status #{packages_list}))
+      return false if status != 0
+
+      status_of_pkg = output.split("\n").grep(/^Status:/).first
+      if status_of_pkg.include?('installed')
+        return true
+      end
+
+      false
```

Comment:
```suggestion
      status_of_pkg.include?('installed')
```

This returns a boolean so you don't need to do anymore than call it here.